image/templates/helm/stackrox-central/templates/_central_setup.tpl

```diff
@@ -52,26 +52,35 @@
     Central's DB PVC config setup
   */}}
 {{ $dbVolumeCfg := dict }}
+{{ $dbBackupVolumeCfg := dict }}
 {{ if not $centralDBCfg.external }}
 {{ if $centralDBCfg.persistence.none }}
   {{ include "srox.warn" (list $ "You have selected no persistence backend. Every deletion of the StackRox Central DB pod will cause you to lose all your data. This is STRONGLY recommended against.") }}
   {{ $_ := set $dbVolumeCfg "emptyDir" dict }}
+  {{ $_ := set $dbBackupVolumeCfg "emptyDir" dict }}
 {{ end }}
 {{ if $centralDBCfg.persistence.hostPath }}
   {{ if not $centralDBCfg.nodeSelector }}
     {{ include "srox.warn" (list $ "You have selected host path persistence, but not specified a node selector. This is unlikely to work reliably.") }}
   {{ end }}
   {{ $_ := set $dbVolumeCfg "hostPath" (dict "path" $centralDBCfg.persistence.hostPath) }}
+  {{ $_ := set $dbBackupVolumeCfg "hostPath" (printf "%s/backup" (dict "path" $centralDBCfg.persistence.hostPath)) }}
```

Comment:
So AFAICT the same directory will be visible at `/var/lib/postgresql/data/backup` (via the `disk` volume) as well as `/var/lib/postgresql/backup` (via the `backup` volume), right? Can you confirm that we're not going to attempt to copy `/var/lib/postgresql/data` into `/var/lib/postgresql/backup` at any point? I think that might cause a runaway copy loop...

Reply:
Yeah, it's going to be `/var/lib/postgresql/data` and `/var/lib/postgresql/data/backup`, I assume that's what you mean? Agree, this should be `/var/lib/postgresql/data` and `/var/lib/postgresql/backup`. In fact there was something similar in the PVC case -- I have fixed that, but haven't applied the fix for hostPath case.

Reply:
I've changed it, but I'm somewhat confused about `hostPath` in general now. Some places claim it has to be a string (`config-shape.yaml`, `values-public.yaml.example.htpl`), but it's clearly an object with the key `path` in it. Moreover when I experiment with centrals setting this value, the central-db deployment is not getting created, all I did was adding `spec.central.db.persistence.hostPath.path` -- do I miss anything?